Fixed a typo in the 2.1 release notes

mlorant opened the pull request:
Straightforward, just a word duplicated by mistake I guess

felixxm commented:
I found _"the the"_ duplication also in:
 - _"internals/contributing/writing-code/coding-style.txt"_ and
 - _"ref/contrib/gis/gdal.txt"_.

Can you fix them?

jschneier commented:
`grep -r "\bthe the\b" *` returns the following for me:

```
django/test/testcases.py:        Assert that expected_message is found in the the message of a raised
django/utils/tree.py:        """Return the the number of children this node has."""
django/contrib/gis/db/backends/base/features.py:    # Can the the function be applied on geodetic coordinate systems?
docs/releases/2.1.txt:* You can now :ref:`override the the default admin site
docs/releases/1.9.txt:Direct assignment now relies on the the new
docs/releases/1.8.txt:  ``'username'``, using the the ``'unique'`` key in its
docs/releases/1.11.txt:  than the the old ``http://openlayers.org`` source. They are also updated to
docs/releases/1.11.txt:  to the the ordering of each column in the index.
docs/ref/contrib/gis/gdal.txt:        name. The default value of the the raster name is the name of the source
docs/internals/contributing/writing-code/coding-style.txt:* Use single quotes for strings, or a double quote if the the string contains a```

felixxm commented:
I've prepared separate PR with removing duplicated words from comments #10192.

mlorant commented:
@felixxm Thanks for the "code" commit part. I've updated the commit and its message to reflect the fact it covers more than release notes now. 

Also added two occurrences found:

```
$ egrep -nr "(\b[a-zA-Z]+) \1\b" docs/ | grep -v ".svg" | less
[...]
docs/ref/csrf.txt:554:a login or if they log in in a different browser tab.
docs/topics/auth/default.txt:1450:        allows inactive users, such as as
[...]
```
